server/core/storage.go

```diff
@@ -362,27 +362,44 @@ func (s *Storage) Close() error {
 }
 
 // SaveGCSafePoint saves new GC safe point to storage.
-func (s *Storage) SaveGCSafePoint(safePoint uint64) error {
-	key := path.Join(gcPath, "safe_point")
+func (s *Storage) SaveGCSafePoint(serviceID string, safePoint uint64, ttl int64) error {
```

Comment:
can it guarantee backward compatibility?

Reply:
Yes, the gRPC API is compatible

Reply:
Will tidb use a serviceID? if so,  the key `xxxx/safepoint` still exists after upgrade and it does not be pushed any more.

Reply:
I was not doubt about API capatibility but the old data. After upgrade PD from old version to new version. will the old saved safePoint get lost?